app/helpers/report_helper.rb

```diff
@@ -0,0 +1,20 @@
+module ReportHelper
+  def quarters_to_date_ranges(quarters)
+    quarters.map! do |quarter_string|
+      quarter = quarter_string.match(/(\d) (\d{4})/)
```

Comment:
Interested to know why this is in a helper rather than in `ReportPresenter`..? 🤔